src/app/components/headerbar/headerbar.css

```diff
@@ -0,0 +1,5 @@
+.headerbar--translucent, .headerbar--translucent windowhandle {
```

Comment:
For future reference: .flat already does exactly what you wanted to do here.

Reply:
oh okay did not know that, I'll fix it! do you have a reference of css classes you could share?

Reply:
Libadwaita demo has a section on style classes.